src/managers/GeneralManager.ts

```diff
@@ -97,6 +97,15 @@ class GeneralManager {
     public getPlayers(): Nullable<PlayerInformation[]> {
         return this.data?.players || null;
     }
+
+    public getSeed(): string {
+        const session: Nullable<SessionInformation> = this.getSession();
+        if (session === null) {
+            throw new Error("Couldn't get Session");
+        } else {
+            return session.seed;
+        }
+    }
```

Comment:
I would suggest moving that method method into the session manager as it is session specific.

Reply:
Little advice on the side to make it fancier: instead of manually checking if session is null and then error you could just "assert(session);" and then return session.seed;